src/FormItem.tsx

```diff
@@ -40,6 +40,15 @@ export const FormItem = <TFieldValues extends FieldValues = FieldValues>({
           isValidElement(child) &&
           cloneElement(child, {
             ...field,
+            //@ts-expect-error onChange type safe is not necessary here
+            onChange: (...params) => {
+              child.props.onChange && child.props.onChange(...params);
+              field.onChange(...params);
+            },
+            onBlur: () => {
```

Comment:
I assume you should remove `onBlur` here cuz ant design haven't provided such prop (for example, [Input Component](https://ant.design/components/input))